triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -29,6 +29,7 @@ import com.daml.lf.data.Ref
 import com.daml.lf.data.Ref._
 import com.daml.lf.data.ScalazEqual._
 import com.daml.lf.data.Time.Timestamp
+import com.daml.lf.engine.trigger.Runner.triggerUserState
```

Comment:
I'm a bit surprised that you have to (or even that you can) import `triggerUserState` from _this very file_, why is that?

Reply:
`triggerUserState` should be private (as we have no intention of anyone else using it) - so I'll push out that small change.

That said, `triggerUserState` is defined in the `Runner` classes companion object. To access this from an instance of the `Runner` class, we need to either import it (as we do here) or use the fully qualified path (i.e. `Runner.triggerUserState`). Does that answer your question?

Reply:
aha, I'd missed that this file defines both `Runner`-the-class and `Runner`-the-object, thanks for explaining!